### ISSUE TITLE
Ubuntu support for DeepSea stage 0

### ISSUE THREAD

**Issue by noelmcloughlin** [saltstack-formulas/deepsea-formula#5]
Use `pkgrepo.managed` so DeepSea works on Ubuntu.  

**Workaround:**
```
sudo -s
wget -q -O- 'https://download.ceph.com/keys/release.asc' | sudo apt-key add -
OK
echo deb https://download.ceph.com/debian-luminous/ $(lsb_release -sc) main | sudo tee /etc/apt/sources.list.d/ceph.list
```

**Comment by noelmcloughlin** [comment]
Can use `ceph.repo` state: https://github.com/saltstack-formulas/ceph-formula/pull/17

**Comment by noelmcloughlin** [comment]
Resolved by #7

**Comment by noelmcloughlin** [comment]
Because `ceph.repo` handles this requirement, this formula will not.  need to update README
